specs/GetEnvironment.md

```diff
@@ -0,0 +1,105 @@
+# Background
+When the WebView2 team was making design changes to the [WebResourceRequested API](https://github.com/MicrosoftEdge/WebView2Feedback/wiki/WebResourceRequested-API-Review-Spec) for .NET, we realized it was not intutive to create the WebView2 Environment. Thus it was not straightforward to use the CreateWebResourceResponse off the WebView2 Environment.
```

Comment:
Instead of '...it was not intutive to create the WebView2 Environment', I'd want to mention both that its not ideal to force all end devs to keep references back to their Environment from their CoreWebView2 event handlers and also that in the case of WPF, WinForms, and WinUI3.0 the UI framework can create the Environment internally with no easy way for the end developer to obtain a reference to it. Providing a reference to the Environment off of the CoreWebView2 solves both of those problems.